_dashboards/install/plugins.md

```diff
@@ -20,25 +20,31 @@ If you don't want to use the all-in-one installation options, you can install th
 
 ## Plugin compatibility
 
-| Plugin Name | OpenSearch Dashboards version |
-| :--- | :--- |
-| alertingDashboards | 1.0.0 or newer |
-| anomalyDetectionDashboards | 1.0.0 or newer |
-| ganttChartDashboards | 1.0.0 or newer |
-| indexManagementDashboards | 1.0.0 or newer |
-| notebooksDashboards | 1.0.0 or newer |
-| notificationsDashboards | 2.0.0.0 or newer |
-| observabilityDashboards | 2.0.0.0 or newer |
-| queryWorkbenchDashboards | 1.0.0 or newer |
-| reportsDashboards | 1.0.0 or newer |
-| securityDashboards | 1.0.0 or newer |
-| traceAnalyticsDashboards | 1.0.0 or newer |
+Major, minor, and patch plugin versions must match OpenSearch major, minor, and patch versions in order to be compatible. For example, plugins versions 2.3.0.x work only with OpenSearch 2.3.0.
+{: .note}
+
+The following table lists available OpenSearch Dashboards plugins.
+
+| Plugin Name | Repository |Earliest Available Version |
+| :--- | :--- | :--- |
+| Alerting | [alerting-dashboards-plugin](https://github.com/opensearch-project/alerting-dashboards-plugin) | 1.0.0 |
+| Anomaly Detection | [anomaly-detection-dashboards-plugin](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin) | 1.0.0 |
+| Search Relevance | [dashboards-search-relevance](https://github.com/opensearch-project/dashboards-search-relevance) | 2.4.0 |
+| Gantt Chart | [gantt-chart](https://github.com/opensearch-project/dashboards-visualizations/tree/main/gantt-chart) | 1.0.0 |
+| Index Management | [index-management-dashboards-plugin](https://github.com/opensearch-project/index-management-dashboards-plugin) | 1.0.0 |
+| Notebooks | [dashboards-notebooks](https://github.com/opensearch-project/dashboards-notebooks) | 1.0.0 |
+| Notifications | [notifications](https://github.com/opensearch-project/notifications) | 2.0.0 |
+| Observability | [dashboards-observability](https://github.com/opensearch-project/observability/tree/main/dashboards-observability) | 2.0.0 |
+| Query Workbench | [query-workbench](https://github.com/opensearch-project/sql/tree/main/workbench) | 1.0.0 |
+| Reports | [dashboards-reports](https://github.com/opensearch-project/dashboards-reports) | 1.0.0 |
+| Security Analytics | [security-analytics-dashboards-plugin](https://github.com/opensearch-project/security-analytics-dashboards-plugin)| 2.4.0 |
+| Security | [security-dashboards-plugin](https://github.com/opensearch-project/security-analytics-dashboards-plugin) | 1.0.0 |
 
 ## Prerequisites
 
 - A compatible OpenSearch cluster
 - The corresponding OpenSearch plugins [installed on that cluster]({{site.url}}{{site.baseurl}}/opensearch/install/plugins/)
-- The corresponding version of [OpenSearch Dashboards]({{site.url}}{{site.baseurl}}/) (e.g. OpenSearch Dashboards 1.0.0 works with OpenSearch 1.0.0)
+- The corresponding version of [OpenSearch Dashboards]({{site.url}}{{site.baseurl}}/) (for example, OpenSearch Dashboards 1.0.0 works with OpenSearch 1.0.0)
```

Comment:
I would change this to a more recent example, that includes at least a minor version:

```suggestion
- The corresponding version of [OpenSearch Dashboards]({{site.url}}{{site.baseurl}}/) (for example, OpenSearch Dashboards 2.1.0 works with OpenSearch 2.1.0)
```